index.html

```diff
@@ -3046,41 +3074,30 @@ <h3>Verify Proof</h3>
 the `proof` value removed.
           </li>
           <li>
-Let <var>transformedData</var> be the result of
-<a href="#dfn-transformation">transforming</a> the <var>unsecuredDocument</var>
-according to a <a>transformation algorithm</a> associated with the
-<a>cryptographic suite</a> specified in <var>proof</var> and the
-<var>options</var> parameters provided as inputs to the algorithm.
-The type of <a>cryptographic suite</a> is specified by the
-<var>proof.type</var> value and MAY be further described by
-<a>cryptographic suite</a>-specific properties expressed in <var>proof</var>.
+Let <var>transformedData</var> be the result of running |suite|'s'
+[=cryptographic suite/transform=] algorithm on the <var>unsecuredDocument</var>.
         </li>
           <li>
-Let <var>hashData</var> be the result of
-<a href="#dfn-hashing">hashing</a> the <var>transformedData</var>
-according to a <a>hashing algorithm</a> associated with the
-<a>cryptographic suite</a> specified in the <var>proof</var> and
-<var>options</var> parameters provided as inputs to the algorithm.
+Let <var>hashData</var> be the result of running |suite|'s'
+[=cryptographic suite/hash=] algorithm on the <var>transformedData</var>.
           </li>
           <li>
-Let <var>isProofVerified</var> be the result of running the
-<a>proof verification algorithm</a> associated with the
-<a>cryptographic suite</a> with the <var>hashData</var> and <var>options</var>
-parameters provided as inputs to the algorithm.
+Let |isProofVerified:valid or invalid| be the result of running |suite|'s
+[=cryptographic suite/verify proof=] algorithm on <var>hashData</var> and <var>proof</var>.
           </li>
           <li>
-If <var>options</var>.<var>domain</var> is set and it does not match
+If <var>domain</var> was given, and it does not match
```

Comment:
"match" is a yellow flag in my spec reviews. Usually it should be "equal", but sometimes people mean a different comparison algorithm that needs to be specified. That fix should go in a different PR.

Reply:
The intention here was "equal" ... as in string equality. Can fix in a future iteration as an editorial fix.